### PR TITLE
clean path in HTMLHelper:: & prevent duplicate /

### PR DESCRIPTION
**Pull request by N6REJ**
Pull Request for Issue # .
#19048

### Summary of Changes
cleaned all paths to ensure platform proper styling with no mixed platform cases.
no extra '/' in path

### Testing Instructions
put preceding '/' in file in HTMLHelper:: call like below..
if($customCSS){ HTMLHelper::_('stylesheet', '/custom.css', $HTMLHelperOptions); }

### Expected result
'<link'> or <script> is created with proper pathing

### Actual result
// between base and filename.
Mixed case of / and \ in path

### Documentation Changes Required
none.

Sorry to have to keep redoing this.. I suck @ git!

### CODE REVIEW

**Comment by N6REJ** [comment]
@viocassel would you retest please if needed.
@brianteeman I  triple checked there is only one file this time.

**Comment by brianteeman** [comment]
> if($customCSS){ HTMLHelper::_('stylesheet', '/custom.css', $HTMLHelperOptions); }

That obviously won't work as a single line as the variables are unique to your template and not part of core.

Instead I modified this line protostar
`JHtml::_('stylesheet', 'user.css', array('version' => 'auto', 'relative' => true));
`
to 
`JHtml::_('stylesheet', '/user.css', array('version' => 'auto', 'relative' => true));
`
and created a user.css file in the template css folder

After doing that (on my windows system) I can see what I _think_ you are reporting when I view the source

`<link href="/cms/templates/protostar/css//custom.css?3b6e1177ff1cdf549f0087a26d714809" rel="stylesheet" />
 `

But that double slash is not a problem and doesnt prevent custom.css file being loaded. And applying the code change made no difference to the //

In summary I don't believe the problem you experienced is related to this at all

**Comment by N6REJ** [comment]
@brianteeman Clearly the variable was an example, especially considering I said "like" below.
As for the issue it may have been cured in the 2yrs since I initially created the issue/pr.. I'll test for the issue again this morning.

**Comment by brianteeman** [comment]
Please provide a clear example that can be used to show the problem you are trying to solve.

**Comment by brianteeman** [comment]
I have honestly no idea what you were doing with your last commit but its now virtually impossible to see what the code is that you changed - plus there is no way that the changes here are an improvement in any way.

/me cant help but feel you are wasting your time on a non-issue
